usaspending_api/search/delta_models/award_search.py

```diff
@@ -0,0 +1,96 @@
+award_search_sql_string = r"""
+CREATE OR REPLACE TABLE {DESTINATION_TABLE} (
+    treasury_account_identifiers ARRAY<INTEGER>,
+    award_id LONG NOT NULL,
+    category STRING,
+    type STRING,
+    type_description STRING,
+    generated_unique_award_id STRING,
+    display_award_id STRING,
+    update_date TIMESTAMP,
+    piid STRING,
+    fain STRING,
+    uri STRING,
+    award_amount NUMERIC(23, 2),
+    total_obligation NUMERIC(23, 2),
+    description STRING,
+    total_obl_bin STRING,
+    total_subsidy_cost NUMERIC(23, 2),
+    total_loan_value NUMERIC(23, 2),
+    recipient_hash BINARY,
```

Comment:
We had some discussion yesterday about whether we should convert `UUID` fields to a `STRING` or `BINARY`. I think we landed on `STRING`, but I'll double check

Reply:
Alright, fixed